owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.kt

```diff
@@ -1622,8 +1622,8 @@ class FileDisplayActivity : FileActivity(),
 
             FileListOption.SHARED_BY_LINK -> {
                 if (previousFileListOption != newFileListOption || initialState) {
-                    val rootFolderForShares = storageManager.getRootSharesFolder()
-                    val personalFolder = storageManager.getRootPersonalFolder()
+                    val rootFolderForShares = storageManager?.getRootSharesFolder()
+                    val personalFolder = storageManager?.getRootPersonalFolder()
```

Comment:
Looking at the report, it seems that the same is happening in the line 1642, where `storageManager` is used as well

Reply:
In relation to the first case of error. We are going to do the following approach: In the `getStorageManager` getter of the `BaseActivity`, we are going to put a condition whether `mStorageManager` is `null`. In this case we will create a new instance of `FileDataStorageManager` using the current account.